qadence/mitigations/analog_zne.py

```diff
@@ -85,7 +207,7 @@ def mitigate(
     mitigated_exp = analog_zne(
         backend_name=backend_name,
         circuit=circuit,
-        observable=observable,
+        observables=observables,
```

Comment:
we always refer to observable in the singular case, even if we pass multiple. lets keep it?

Reply:
Right, done.

Reply:
Hm, I am not sure it makes sense to singularise a list name though.